tools/loader.cc

```diff
@@ -23,10 +23,14 @@ int main(int argc, char **argv) {
   std::string content((std::istreambuf_iterator<char>(input)),
                       std::istreambuf_iterator<char>());
 
-  uint64_t *ebpf_instructions = NULL;
+  uint8_t *ebpf_instructions = NULL;
+  uint8_t *func = NULL;
+  uint8_t *btf = NULL;
   uint64_t array_length = 0;
+  uint64_t func_length = 0;
+  uint64_t btf_length = 0;
   EncodeEBPF(content.data(), content.length(), &ebpf_instructions,
```

Comment:
Could you add a TODO to fix the EncodeEBPF program to return a serialized proto? I think it has a bit too many parameters but I don't want us to spend a lot of time on this now :)

Reply:
Done, already fixed cuz of the change to pass the proto to `load_ebpf_program`